v4/api/dto/users/UserResources.java

```diff
@@ -0,0 +1,72 @@
+package com.equinix.openapi.fabric.v4.api.dto.users;
```

Comment:
What is different between this and the code generated files? Why are there so many hardcopies? Can we leverage the code generation more?

Reply:
unfortunately no, I explained in another comment

Reply:
fixed problem by adjusting module structure